---- Minecraft Crash Report ----
// My bad.

Time: 25.03.25 22:52
Description: Exception generating new chunk

java.lang.ArrayIndexOutOfBoundsException: 16
	at net.minecraft.world.chunk.Chunk.func_150807_a(Chunk.java:571)
	at com.github.krz19.WorldGen.CitiesGenChunkProvider.fillChunkWithStone(CitiesGenChunkProvider.java:52)
	at com.github.krz19.WorldGen.CitiesGenChunkProvider.provideChunk(CitiesGenChunkProvider.java:36)
	at net.minecraft.world.gen.ChunkProviderServer.originalLoadChunk(ChunkProviderServer.java:172)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:131)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:101)
	at net.minecraft.server.MinecraftServer.initialWorldChunkLoad(MinecraftServer.java:265)
	at net.minecraft.server.integrated.IntegratedServer.loadAllWorlds(IntegratedServer.java:78)
	at net.minecraft.server.integrated.IntegratedServer.startServer(IntegratedServer.java:92)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.world.chunk.Chunk.func_150807_a(Chunk.java:571)
	at com.github.krz19.WorldGen.CitiesGenChunkProvider.fillChunkWithStone(CitiesGenChunkProvider.java:52)
	at com.github.krz19.WorldGen.CitiesGenChunkProvider.provideChunk(CitiesGenChunkProvider.java:36)

-- Chunk to be generated --
Details:
	Location: -11,18
	Position hash: 81604378613
	Generator: RandomLevelSource
Stacktrace:
	at net.minecraft.world.gen.ChunkProviderServer.originalLoadChunk(ChunkProviderServer.java:172)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:131)
	at net.minecraft.world.gen.ChunkProviderServer.loadChunk(ChunkProviderServer.java:101)
	at net.minecraft.server.MinecraftServer.initialWorldChunkLoad(MinecraftServer.java:265)
	at net.minecraft.server.integrated.IntegratedServer.loadAllWorlds(IntegratedServer.java:78)
	at net.minecraft.server.integrated.IntegratedServer.startServer(IntegratedServer.java:92)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_422, BellSoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), BellSoft
	Memory: 549233728 bytes (523 MB) / 703594496 bytes (671 MB) up to 2837446656 bytes (2706 MB)
	JVM Flags: 0 total; 
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 90
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 4 mods loaded, 4 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAA	FML{7.10.99.99} [Forge Mod Loader] (forgeBin-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJAA	Forge{10.13.4.1614} [Minecraft Forge] (forgeBin-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJAA	FurnitureMod{0.0.1} [Furniture Mod] (roadmod-1.0.jar) 
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 0 / 8; []
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'